pocApp/new_tasks.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+from random import choice
+from celery import Celery
+from celery import shared_task, task
+from celery.decorators import periodic_task
+from libnmap.process import NmapProcess
+import xml.dom.minidom
+import sys
+import logging
+import datetime
+
+from pocApp.models import MyCertsData
+from orion_integration.lib import OrionSslNode
+
+logger = logging.Logger('catch_all')
+
+@shared_task
+def go():
+    return True
+    for i in range(10):
+        print ("I am @ record:%d " % (i))
+
+@task()
+def getnmapdata():
+    rc = "success"
+    nodeList = OrionSslNode.ip_addresses()
```

Comment:
node_list (not nodeList) must be populated using the nodes method, not the ip_addresses method. it must include both orionnode_orionid and orionapmapplication_orionid